src/gc/gcconfig.cpp

```diff
@@ -95,7 +95,7 @@ bool ParseGCHeapAffinitizeRanges(const char* cpu_index_ranges, AffinitySet* conf
 
     if (cpu_index_ranges != NULL)
     {
-        const char* number_end;
+        const char* number_end = nullptr;
```

Comment:
This is fixing uninitialized variable warning reported by VS2019. Looks like a real bug.

Reply:
Yes, it is a bug I've introduced during refactoring of this function. The original version was always setting the number_end via strtoul.
As for the fix, you could also just initialize the number_end with cpu_index_ranges. Then the success check could stay as it was. The semantics of the number_end is that it points after after the last parsed number. So such change would keep that semantics as at the init time is the first character, as there were no numbers parsed yet.